src/util.cpp

```diff
@@ -387,6 +388,7 @@ void ArgsManager::WarnForSectionOnlyArgs()
     // if it's okay to use the default section for this network, don't worry
     if (m_network == CBaseChainParams::MAIN) return;
 
+    LOCK(cs_args);
```

Comment:
Just take the lock in the very first line of this method, or is there a specific reason that `m_network` shouldn't be protected?